docs/FIXTURE.md

```diff
@@ -0,0 +1,284 @@
+```
+
+## Attributes - `akeneo_attributes`
+
+This fixture allow you to trigger the `akeneo:import:attributes` command.
+
+You can specify the batch size, if you allow parallel import, and max concurrency. 
+
+```yaml
+# config/packages/sylius_fixtures.yaml
+
+sylius_fixtures:
+    suites:
+        app:
+            fixtures:
+                akeneo_attributes: 
+                    options:
+                        batch_size: 100
+                        allow_parallel: true
+                        max_concurrency: 4
+```
+
+If you have too many attributes, you can also provide a custom filter to reduce the number of attributes you want to import.
+
+```yaml
+# config/packages/sylius_fixtures.yaml
+
+sylius_fixtures:
+    suites:
+        app:
+            fixtures:
+                akeneo_attributes: 
+                    options:
+                        custom:
+                            search:
+                                code: 
+                                    - 
+                                        operator: 'IN'
+                                        value: 
+                                            - 'attribute_code'
+                                            - 'another_attribute_code'
+```
+Inside `custom` node option, you can provide any query parameter supported by [Akeneo API](https://api.akeneo.com/documentation/filter.html#filter-attributes).
+
+## Association Types - `akeneo_association_types`
+
+This fixture allow you to trigger the `akeneo:import:association-type` command.
+
+You can specify the batch size, if you allow parallel import, and max concurrency. 
+
+```yaml
+# config/packages/sylius_fixtures.yaml
+
+sylius_fixtures:
+    suites:
+        app:
+            fixtures:
+                akeneo_association_types: 
+                    options:
+                        batch_size: 100
+                        allow_parallel: true
+                        max_concurrency: 4
+```
+
+> [!NOTE]
+> Given [Akeneo API](https://api.akeneo.com/api-reference.html#Associationtype) don't provide any search filter for association types, 
+> it's not currently possible de reduce the number of association types to import.
+
+## Families - `akeneo_families`
+
+This fixture allow you to trigger the `akeneo:import:families` command.
+
+You can specify the batch size, if you allow parallel import, and max concurrency.
+
+```yaml
+# config/packages/sylius_fixtures.yaml
+
+sylius_fixtures:
+    suites:
+        app:
+            fixtures:
+                akeneo_families: 
+                    options:
+                        batch_size: 100
+                        allow_parallel: true
+                        max_concurrency: 4
+```
+
+You can also provide a custom filter to reduce the number of families you want to import.
+
+Inside `custom` node option, you can provide any query parameter supported by [Akeneo API](https://api.akeneo.com/documentation/filter.html#filter-on-product-model-properties).
```

Comment:
```suggestion
Inside `custom` node option, you can provide any query parameter supported by [Akeneo API](https://api.akeneo.com/documentation/filter.html#filter-families).
```

Reply:
No, we call product-model api here

Reply:
Logic, or not 😂 
Ok, we build families from products datas.